### ISSUE TITLE
 Error building application using circleci（turtle-cli-example）

### ISSUE THREAD

**Issue by panlei09** [expo/turtle#137]
Error message: Sep 5 09:49:39 turtle[191] INFO: Configuring Android SDK, this may take a while
  Platform: "android"
  buildPhase: "setting up environment"
Sep 5 09:50:25 turtle[191] INFO: Android SDK installed and configured successfully
  Platform: "android"
  buildPhase: "setting up environment"
Sep 5 09:50:25 turtle[191] INFO: Downloading Android NDK
  Platform: "android"
  buildPhase: "setting up environment"
Downloading [] 0% 0.0sdownloading [] 12% 7.3sdownloading [] 24% 6.1sdownloading [] 37% 5.0sdownloading [] 48% 4.2sdownloading [] 61% 3.2sdownloading [] 74% 2.1sdownloading [] 86% 1.1sdownloading [ ] 98% 0.1sSep 5 09:50:33 turtle[191] INFO: Decompressing Android NDK
  Platform: "android"
  buildPhase: "setting up environment"
Received 'killed' signal

**Comment by wiloke1** [comment]
same

**Comment by dbyst** [comment]
Same

**Comment by wiloke1** [comment]
@panlei09 Need someone to help with this problem

**Comment by dsokal** [comment]
Hello, 
could you please tell me what `resource_class` have you set? In the repository with the example you mentioned we use  `resource_class: xlarge`. If I recall correctly any lower resource class wasn't enough for building an android project and was resulting in the error you posted.

**Comment by dsokal** [comment]
Hi, 
I'm closing the issue because of inactivity. Please let me know if the problem still exists.

**Comment by brayanL** [comment]
I have the same error:
```
Nov 12 17:49:21 turtle[197] WARN:  There is a new version of turtle-cli available (0.12.8).
You are currently using turtle-cli 0.12.7
Run `npm install -g turtle-cli` to get the latest version
  platform: "android"
Nov 12 17:49:23 turtle[197] INFO:  Downloading Android SDK
  platform: "android"
  buildPhase: "setting up environment"
downloading [] 0% 0.0sdownloading [] 64% 0.5sNov 12 17:49:25 turtle[197] INFO:  Decompressing Android SDK
  platform: "android"
  buildPhase: "setting up environment"
Nov 12 17:49:28 turtle[197] INFO:  Configuring Android SDK, this may take a while
  platform: "android"
  buildPhase: "setting up environment"
Nov 12 17:50:05 turtle[197] INFO:  Android SDK installed and configured successfully
  platform: "android"
  buildPhase: "setting up environment"
Nov 12 17:50:05 turtle[197] INFO:  Downloading Android NDK
  platform: "android"
  buildPhase: "setting up environment"
downloading [] 0% 0.0sdownloading [] 2% 39.3sdownloading [] 4% 38.8sdownloading [] 7% 38.6sdownloading [] 10% 35.9sdownloading [] 13% 33.5sdownloading [] 15% 33.1sdownloading [] 17% 32.0sdownloading [] 21% 29.8sdownloading [] 24% 28.2sdownloading [] 27% 26.5sdownloading [] 30% 25.4sdownloading [] 33% 23.5sdownloading [] 36% 22.3sdownloading [] 40% 20.7sdownloading [] 43% 19.6sdownloading [] 46% 18.5sdownloading [] 49% 17.1sdownloading [] 53% 15.8sdownloading [] 56% 14.5sdownloading [] 60% 13.3sdownloading [] 63% 12.1sdownloading [] 66% 10.9sdownloading [] 70% 9.8sdownloading [] 73% 8.6sdownloading [] 76% 7.7sdownloading [] 79% 6.6sdownloading [] 83% 5.3sdownloading [] 87% 4.1sdownloading [] 90% 3.0sdownloading [] 94% 1.8sdownloading [] 97% 0.7sNov 12 17:50:37 turtle[197] INFO:  Decompressing Android NDK
  platform: "android"
  buildPhase: "setting up environment"
Received 'killed' signal
```

What is the solution?

**Comment by wiloke1** [comment]
@dsokal same error. I using `config.yml` here:
https://github.com/expo/turtle-cli-example/blob/master/.circleci/config.yml
